contracts/core/diaspore/cosigner/Collateral.sol

```diff
@@ -177,6 +179,8 @@ contract Collateral is Ownable, Cosigner, ERC721Base {
             _loanId,
             _oracleData
         );
+
+        return true;
```

Comment:
If no marginCall or duePayment was executed, the claim did nothing and it should return false.

Reply:
This is true, in the case of only the margin call was executed this function should be return true?